### PR TITLE
Migrate article: Technical Emails #1080

### PR DESCRIPTION
**Pull request by dragonlancewu**
Migrated help article : Technical Emails

### CODE REVIEW

**Review by anelook — DISMISSED** [review]
Thank you @dragonlancewu for migrating this article! I left one comment related to the title of the article to align with other materials in this section.

File: docs/platform/howto/technical-emails.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Technical Emails
```
[the remark this answers is not in the document]
Comment (dragonlancewu) [reply docs/platform/howto/technical-emails.rst:1]: Thank you for your suggestion, I updated the title.

**Comment by runwuf** [comment]
Thank you @laysauchoa and @anelook for reviewing this PR.
@dragonlancewu and I have replaced the screenshot and added the updates as per review.

**Review by laysauchoa — APPROVED** [review]
I like it. Thank you for this contribution!

**Pull request merged**